internal/restinterface/tls/tls.go

```diff
@@ -67,7 +73,11 @@ type HasCertificate struct {
 	IsSetCert bool
 }
 
-func SetPSKHandler(h PSKHandler) {
+// func SetPSKHandler(h PSKHandler) {
```

Comment:
Better to leave any comment here?

Reply:
@MoonkiHong Thank you, I accepted all the corrections!